src/components/seeds/database/index.tsx

```diff
@@ -407,33 +410,111 @@ export default function Database(props: DatabaseProps): JSX.Element {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
   const getHeaderButtons = () => (
     <>
-      <Button
-        id='edit-columns'
-        label={strings.ADD_COLUMNS}
-        onClick={onOpenEditColumnsModal}
-        priority='secondary'
-        type='passive'
-        size='medium'
-        className={classes.buttonSpc}
-      />
-      <Button
-        id='download-report'
-        label={strings.DOWNLOAD_AS_REPORT}
-        onClick={onDownloadReport}
-        priority='secondary'
-        type='passive'
-        size='medium'
-        className={classes.buttonSpc}
-      />
+      <ButtonMUI
+        aria-describedby={id}
+        variant='contained'
+        onClick={handleClick}
+        sx={{
+          borderRadius: '40px',
+          width: '40px',
+          minWidth: '40px',
+          color: '#0067C8',
+          background: '#fff',
+          border: '2px solid #0067C8',
+          padding: 0,
+          fontSize: '20px',
+          marginRight: 1,
+        }}
+      >
+        ...
+      </ButtonMUI>
+      <Popover
+        id={id}
+        open={open}
+        anchorEl={anchorEl}
+        onClose={handleClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'left',
+        }}
+      >
+        <Box>
+          <Button
+            id='edit-columns'
+            label={strings.ADD_COLUMNS}
+            onClick={onOpenEditColumnsModal}
+            priority='secondary'
+            type='passive'
+            size='medium'
+            className={classes.buttonSpc}
+          />
+        </Box>
+        <Box>
+          <Button
+            id='download-report'
+            label={strings.DOWNLOAD_AS_REPORT}
+            onClick={onDownloadReport}
+            priority='secondary'
+            type='passive'
+            size='medium'
+            className={classes.buttonSpc}
+          />
+        </Box>
+      </Popover>
+      <Box paddingRight={1} display='inline'>
+        <Button label={strings.IMPORT} size='medium' onClick={importAccessions} priority='secondary' />
+      </Box>
     </>
   );
 
   return (
     <LocalizationProvider dateAdapter={AdapterDateFns}>
+      {selectedOrgInfo.selectedFacility && (
+        <ImportAccessionsModal
+          open={openImportModal}
+          onClose={() => setOpenImportModal(false)}
+          facility={selectedOrgInfo.selectedFacility}
+        />
+      )}
       {organization && (
-        <SelectSeedBankModal organization={organization} open={selectSeedBankModalOpen} onClose={onSeedBankSelected} />
+        <>
+          <SelectSeedBankModal
+            organization={organization}
+            open={selectSeedBankModalOpen}
+            onClose={onSeedBankSelected}
+          />
+          <SelectSeedBankModal
+            organization={organization}
+            open={selectSeedBankForImportModalOpen}
+            onClose={onSeedBankForImportSelected}
+          />
```

Comment:
lets try to use one modal? we should be able to use the correct callbacks based on which flow triggered the modal?